src/reducers/userDataReducer/reducer.js

```diff
@@ -39,6 +39,12 @@ export const useUserDataReducer = () => {
       case "SETWISHLIST": {
         return { ...state, user: { ...state.user, wishlist: payload } };
       }
```

Comment:
You can write SET_WISHLIST instead of SETWISHLIST , so it will be more readable.